src/components/ValidatedInput/ValidFieldPassword.scss

```diff
@@ -37,27 +37,62 @@
     }
   }
 
-  .level {
-    position: absolute;
-    margin-top: 5px;
-    right: 0;
-    font-size: $font-size-h5;
-    text-transform: capitalize;
-
-    &.poor {
-      color: $color--error--primary;
-    }
-    &.weak {
-      color: $color--warning--primary;
-    }
-    &.good {
-      color: $color--info--primary;
+  .password-strength-meter {
+    .strength-meter-bar {
+      display: grid;
+      grid-gap: 5px;
+      grid-auto-flow: column;
+      height: 5px;
+      border-radius: 4px;
+      margin: 10px 0;
+      .bar {
+        background-color: $base-120;
```

Comment:
after taking a look in my local, i think it's too dark and it's hard to distinguish the filled and unfilled bar. could we have a lighter color for the unfilled one?

Reply:
after checking again, i think it's different from the figma design

Reply:
after checking again, i think it's different from the figma design